sno/crs_util.py

```diff
@@ -157,3 +189,12 @@ def normalise_wkt(wkt):
     token_iter = WKTLexer().get_tokens(wkt, pretty_print=True)
     token_value = (value for token_type, value in token_iter)
     return "".join(token_value)
+
+
+def has_standard_authority(crs):
+    """
+    Returns True if this CRS appears to be defined by some standard authority - EPSG, ESRI, or Google.
+    Doesn't check if the CRS actually matches (or roughly matches) the definition specified by that authority.
+    """
+    auth_name, auth_code = parse_authority(crs)
+    return auth_name in ("EPSG", "ESRI") or auth_code == 600613  # GOOGLE
```

Comment:
I think you mean 900913?

```suggestion
    return auth_name in ("EPSG", "ESRI") or auth_code == 900913  # GOOGLE
```

(900913 was later defined as `EPSG:3857`, FWIW)

Reply:
Done